app/views/completed_transaction/_assisted_digital_satisfaction_survey.html.erb

```diff
@@ -1,163 +1,132 @@
-          Satisfied
-        </label>
-        <br />
-        <label class="block-label">
-          <input type="radio" required name="service_feedback[assistance_satisfaction_rating]" id="assistance-neither-satisfied-or-dissatisfied" value="3">
-          Neither satisfied or dissatisfied
-        </label>
-        <br />
-        <label class="block-label">
-          <input type="radio" required name="service_feedback[assistance_satisfaction_rating]" id="assistance-dissatisfied" value="2">
-          Dissatisfied
-        </label>
-        <br />
-        <label class="block-label">
-          <input type="radio" required name="service_feedback[assistance_satisfaction_rating]" id="assistance-very-dissatisfied" value="1">
-          Very dissatisfied
-        </label>
-      </fieldset>
-
-      <br />
-
-      <fieldset>
-        <legend><h2>Is there any way the assistance received could be improved?</h2></legend>
-        <br />
-        <label for="assistance-improvement-comments" class="visuallyhidden">Your comments</label>
-        <textarea name="service_feedback[assistance_improvement_comments]" class="full-size counted" id="assistance-improvement-comments"></textarea>
-        <p id="assistance-improvement-commentscounter" class="hint" aria-live="polite" aria-atomic="false">(Limit is 1200 characters)</p>
-      </fieldset>
-
-      <br />
-    </div>
-  </div>
+  <% government_staff = capture do %>
+    <%= render 'assistance_received', locals: { id_prefix: 'staff_member'} %>
+  <% end %>
 
-  <fieldset>
-    <legend><h2>Overall, how satisfied are you with the online service?</h2></legend>
-    <br />
-    <label class="block-label">
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="service-very-satisfied" value="5">
-      Very satisfied
-    </label>
-    <br />
-    <label class="block-label">
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="service-satisfied" value="4">
-      Satisfied
-    </label>
-    <br />
-    <label class="block-label">
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="service-neither-satisfied-or-dissatisfied" value="3">
-      Neither satisfied or dissatisfied
-    </label>
-    <br />
-    <label class="block-label">
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="service-dissatisfied" value="2">
-      Dissatisfied
-    </label>
-    <br />
-    <label class="block-label">
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="service-very-dissatisfied" value="1">
-      Very dissatisfied
-    </label>
-  </fieldset>
+  <% other_person = capture do %>
+    <%= render 'assistance_received', locals: { id_prefix: 'other'} %>
+  <% end %>
 
-  <br />
+  <% other_input = capture do %>
+    <%= render "govuk_publishing_components/components/input", {
```

Comment:
Shame the styles for this input are being overridden by govuk_template in static. Soon it will be gone!